buildkite/graphql/cluster_queue.graphql

```diff
@@ -0,0 +1,78 @@
+fragment ClusterQueueValues on ClusterQueue {
```

Comment:
Any reason you didn't name this as just `ClusterQueue`?
```suggestion
fragment ClusterQueue on ClusterQueue {
```

Reply:
I'd argue that `Values` makes this more readable; it doesn't fill an entire ClusterQueue.

Reply:
Also for readability - i.e `ClusterQueue on ClusterQueue` -`ClusterQueueValues` being the values returned via said fragment in the response(s) of the query/mutations 👍

Reply:
Ahh yep okay fair enough 👍